README.md

```diff
@@ -144,6 +144,8 @@ TFTP settings are needed.
   </tr>
 </table>
 
+(*) - Foreman 1.14 or older do require DHCP reservation
```

Comment:
Yeah bad wording, fixed. To confirm: Nowdays neither DHCP service nor DHCP reservation is required, PXELinux template will create Anaconda configuration with static IP just fine.